src/drag.js

```diff
@@ -1,5 +1,277 @@
 var syn = require('./synthetic');
 
+
+
+
+// returns the element that exists at a provided x, y coordinate
+// TODO: move this to element.js
+var elementFromPoint = function (point, win) {
+	var clientX = point.clientX;
+	var clientY = point.clientY;
+
+	if(point == null){return null;}
+	
+	if (syn.support.elementFromPage) {
+		var off = syn.helpers.scrollOffset(win);
+		clientX = clientX + off.left; //convert to pageX
+		clientY = clientY + off.top; //convert to pageY
+	}
+	
+	return win.document.elementFromPoint(Math.round(clientX), Math.round(clientY));
+};
+
+
+
+//======================================================================================================
+//  DragonDrop prototype STARTS
+//======================================================================================================
+var DragonDrop = {
+	
+	focusWindow : null,
+	
+	/**
+	* Performs a series of dragStart, dragEnter, dragOver and drop operations to simulate a dragDrop
+	* @param fromElement: element from where drag is started
+	* @param channel: destination element for drop
+	*/
+	dragAndDrop: function(focusWindow, fromPoint, toPoint, duration, callback){
+		this.currentDataTransferItem = null;
+		this.focusWindow = focusWindow;
+	
+		// This would be a series of events to syntesize a drag operation
+		this._MouseOver(fromPoint);
```

Comment:
anyway these can be lowercase?

Reply:
Xenophon is an event tracker tool that I wrote. Its useful to see what events are being received by various interactions. You can run it and record a human's behavior, then run it with Syn/FuncUnit, and compare the two.

We can use Xenophon to increase the accuracy of Syn/FuncUnit event generation.

As-for lowercasing these functions: yes, I will lowercase them and submit that shortly.